repos/system_upgrade/common/actors/verifytargetversion/libraries/verifytargetversion.py

```diff
@@ -0,0 +1,54 @@
+from leapp import reporting
+from leapp.cli.commands import command_utils
+from leapp.exceptions import StopActorExecution
+from leapp.libraries.common.config import get_env
+from leapp.libraries.common.config.version import get_target_version
+from leapp.libraries.stdlib import api
+
+FMT_LIST_SEPARATOR = '\n    - '
+
+
+def process():
+    flavour = get_env('LEAPP_UPGRADE_PATH_FLAVOUR')
+    target_version = get_target_version()
+    supported_target_versions = command_utils.get_supported_target_versions(flavour)
+
+    leapp_unsupported = get_env('LEAPP_UNSUPPORTED', '0') == '1'
```

Comment:
See my comment bellow
```suggestion
    is_leapp_unsupported_set = get_env('LEAPP_UNSUPPORTED', '0') == '1'
```

Reply:
refactored completely